plugins/repository-s3/build.gradle

```diff
@@ -101,6 +101,8 @@ String s3EC2BasePath = System.getenv("amazon_s3_base_path_ec2")
 String s3ECSBucket = System.getenv("amazon_s3_bucket_ecs")
 String s3ECSBasePath = System.getenv("amazon_s3_base_path_ecs")
 
+boolean s3DisableChunkedEncoding = (new Random()).nextBoolean()
```

Comment:
I'd prefer this to be deterministic. You can use `new Random(Long.parseUnsignedLong(project.rootProject.testSeed.tokenize(':').get(0), 16))` to construct a seeded `Random`.